content/index.pl.md

```diff
@@ -0,0 +1,163 @@
+{:toc}
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Wprowadzenie {#intro}
+
+Wytyczne dla dostępności internetowej (WCAG) 2 zostały opracowane w ramach [procesu W3C](/standards-guidelines/w3c-process/) we współpracy z osobami i organizacjami z całego świata, aby zapewnić jednolity wspólny standard dostępności internetowej, który spełnia potrzeby osób, organizacji i rządów na całym świecie.
+
+Dokumenty WCAG wyjaśniają, jak zwiększyć dostępność treści internetowych dla osób z niepełnosprawnościami. „Treść” internetowa ogólnie odnosi się do informacji na stronie internetowej lub w aplikacji internetowej i&nbsp;obejmuje:
+
+-   naturalne informacje, takie jak tekst, obrazy i dźwięki
+-   kod lub znaczniki definiujące strukturę, prezentację itp.
+
+## Dla kogo są WCAG {#for}
+
+WCAG są dla tych, którzy potrzebują standardu technicznego. **To nie jest wprowadzenie do dostępności. Łącza do materiałów wprowadzających można znaleźć w sekcji [“Where should I start?” w FAQ FAQ](/standards-guidelines/wcag/faq/#start).**
+
+WCAG są przeznaczone przede wszystkim dla:
+
+-   twórców treści internetowych (autorów stron, projektanci witryn itp.)
+-   twórców narzędzi do tworzenia stron internetowych (narzędzi autorskich)
+-   twórców narzędzi do oceny dostępności internetowej
+-   innych osób, które potrzebują standardu dostępności internetowej, w tym dostępności aplikacji mobilnych.
+
+Aby zaspokoić potrzeby wielu różnych osób, w tym decydentów, menedżerów, badaczy i innych - istnieje wiele różnych  [[materiałów WAI]](/resources/).
+
+## Co jest w WCAG 2 {#whatis2}
+
+WCAG 2.2 ma 13 wytycznych. Wytyczne są zorganizowane według [4 zasad: postrzegalne, funkcjonalne, zrozumiałe i solidne](https://www.w3.org/WAI/WCAG22/Understanding/intro#understanding-the-four-principles-of-accessibility).
+
+Dla każdej wytycznej istnieją sprawdzalne *kryteria sukcesu*. Kryteria sukcesu są na [trzech poziomach: A, AA oraz AAA](https://www.w3.org/WAI/WCAG22/Understanding/conformance#levels).
+
+Kryteria sukcesu określają „zgodność” z WCAG. Oznacza to, że aby spełnić wymagania WCAG, treść musi spełniać kryteria sukcesu. Szczegóły znajdują się w [WCAG w sekcji Zgodność](https://www.w3.org/TR/WCAG22/#conformance).
+
+Krótkie podsumowanie wytycznych WCAG 2 można znaleźć w artykule **[[WCAG 2 w skrócie]](/standards-guidelines/wcag/glance/)**.
+		
+### Materiały pomocnicze i dodatkowe wskazówki {#supplement}
+
+Poza WCAG, poniższe materiały pomagają zrozumieć i wdrażać WCAG oraz poprawiać dostępność:
+* Krótki przewodnik / Jak spełnić WCAG 2 / Lista kontrolna
+* Objaśnienia WCAG 2
+* Techniki dla WCAG 2
+* Zasady testowania dla WCAG 2
+* Dodatkowe wytyczne
+
+**Zapoznaj się z tymi zasobami WCAG 2 w [dokumentacji WCAG 2](/standards-guidelines/wcag/docs/).**
+
+## WCAG 2.0, 2.1, 2.2 {#versions}
+
+Standardy Wytyczne dla dostępności treści internetowych (WCAG) są stabilne i można się do nich odwoływać,  kiedy są opublikowane jako 'Rekomendacja W3C'.
```

Comment:
Original text: 
> The Web Content Accessibility Guidelines (WCAG) standards are stable and referenceable when they are published as a ‘W3C Recommendation’ web standard.

@zwiastunsw "web standard" seems to be missing in the translation.